filetests/isa/x86/windows_fastcall_x64_unwind.clif

```diff
@@ -160,7 +160,7 @@ ebb0(v0: i64, v1: i64):
 ; nextln:         UnwindCode {
 ; nextln:             offset: 19,
 ; nextln:             op: SmallStackAlloc,
-; nextln:             info: 4,
+; nextln:             info: 3,
```

Comment:
Just so you don't have to read all the code related to UnwindCode as I did: this is the multiple of 8-bytes pushed by the stack allocation; the function in the test is a leaf function, so it matches the criteria to not align the stack frame, thus reducing stack frame allocation.

Reply:
:+1: